common/core/web/keyboard-processor/src/build.sh

```diff
@@ -49,10 +49,5 @@ fi
 # Generates a linkable TS file; defined in resources/build-utils.sh.
 exportEnvironmentDefinitionTS
 
-pushd ../../utils/src
-./build.sh -skip-package-install || fail "Failed to compile Keyman utility function package"
-echo ""
-popd
-
```

Comment:
Notes from online discussion:

Noting https://docs.npmjs.com/misc/scripts and https://github.com/keymanapp/keyman/pull/3146/files#diff-653957a585abd01f82cbb9b7806e8acdR9, any `lerna bootstrap` call will now automatically re-build `web-utils`.  Since all of our Web-oriented build scripts use `verify_npm_setup` - which calls `lerna bootstrap` - they will all also re-build `web-utils` as a result.